test/integration/apiserver/apply/apply_crd_test.go

```diff
@@ -487,6 +487,7 @@ func TestApplyCRDUnhandledSchema(t *testing.T) {
 	var c apiextensionsv1beta1.CustomResourceValidation
 	err = json.Unmarshal([]byte(`{
 		"openAPIV3Schema": {
+            "type": "object",
```

Comment:
I'm suspicious of needing to change this test case... if this was accepted previously, then shouldn't we be fixing CEL to handle this?